airlock/templates/activity.html

```diff
@@ -0,0 +1,25 @@
+{% #card title="Recent Activity" container=True %}
+  {% #table %}
+    {% #table_head class="bg-slate-200" %}
+      {% #table_row %}
+        {% #table_header %}Time{% /table_header %}
+        {% #table_header %}User{% /table_header %}
+        {% #table_header %}Action{% /table_header %}
+        {% #table_header %}Context{% /table_header %}
+      {% /table_row %}
+    {% /table_head %}
+    {% #table_body %}
+      {% for log in activity %}
+        {% #table_row %}
+          {% #table_cell %}{{ log.created_at|date:'Y-m-d H:i' }}{% /table_cell %}
+          {% #table_cell %}{{ log.user }}{% /table_cell %}
+          {% #table_cell %}{{ log.description }}{% /table_cell %}
+          {% #table_cell %}
+            {% if log.path %}{{ log.path }}{% endif %}
+            {% if 'group' in log.extra %}(group: {{ log.extra.group }}){% endif %}
```

Comment:
wdyt about giving group its own column? I realise my local dev audit log isn't going to be representative, but a lot of the events have a group, so my guess is that it would be tidier

<img width="1476" alt="Screenshot 2024-04-26 at 17 44 48" src="https://github.com/opensafely-core/airlock/assets/2082895/cd2fb196-c44d-4eb3-a9a0-bd5bdb9f4433">

Reply:
Yeah, I tried that, it's messy.

For example many workspace audit logs (high volume) have no group

I think we will need to probably specialise these activity logs for each type thing you're looking at.

Reply:
ok, it's a nit we always pick at later if we want

Reply:
Yeah, I just got the most basic version in, mainly so I had *something* to show in the workspace summary!